bin/console

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env php
+<?php
+
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+require_once __DIR__ . '/../app/autoload.php';
+
+$input = new ArgvInput();
+$output = new ConsoleOutput();
+$io = new SymfonyStyle($input, $output);
+
+$io->warning('This command will be executed in the context "admin". If you want to use another context please use "./bin/adminconsole" or "./bin/websiteconsole".');
```

Comment:
This command will be executed in the "admin" context. For the "website" context, run ./bin/websiteconsole